app.server/settings.ts

```diff
@@ -7,24 +7,18 @@ export function getEnvironmentSettings(env: string) {
       dev: true,
       port: 5000,
       dir: 'built/app.client'
-    }
+    };
```

Comment:
There are only a few rules in the code style in the readme.md for the project but one is to not use semicolons. It's probably not enforced very well as there's not tslinting going on. But let's not add them inconsistently; the vast majority of the existing code doesn't use semis.

Reply:
If you want to leave this one to me I could add tslint quite easily.

Reply:
I honestly much prefer semi-colons in the code, it makes it clear when a line should end

Reply:
I always love a good style debate... 😄 But not relevant for an existing project, with existing style guide.

Reply:
Fixed in #50